modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,704 @@
+
+
+=== Spellchecker Pro 3.1.1
+
+The {productname} 6.3 release includes an accompanying release of the **Spellchecker Pro** premium plugin.
+
+**Spellchecker Pro** 3.1.1 includes one fix:
+
+==== Spellchecker Pro could search outside of the {productname} editor for a language match.
+
+For spelling, a word’s language was identified by finding the nearest `lang` attribute anywhere in an ancestor HTML element to the word (ie, any elements that contain the word). 
+
+When {productname} was running in Inline mode, that included HTML ancestors that were provided by the host Browser, which are outside the {productname} editor instance.
+
+If a `lang` attribute existed anywhere in this ancestry that `lang` attribute value overrode all attempts by the user to set the spell checking language for the document.
+
+As of {productname} 6.3, the `lang` attribute inherits the default settings for the {productname} editor body. As a consequence, the editor no longer searches elements external to the editor body when searching for `lang` attributes.
+
+For information on the **Spellchecker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[Spellchecker Pro].
+
+=== Spelling Autocorrect 1.0.1
+
+The {productname} 6.3 release includes an accompanying release of the **Spelling Autocorrect** premium plugin.
+
+**Spelling Autocorrect** 1.0.1 includes one fix:
+
+==== Fixed format changes at the start of sentences when auto-capitalization was on
+
+When **Spelling Autocorrect** was active and had Capitalization enabled (ie had `autocorrect_capitalize: true` set), any inline formatting applied at the beginning of a new sentence, after a full stop (period) but before the start of the word, would be lost when the word was auto-capitalized. Instead the format at the end of the previous sentence was incorrectly applied. 
+
+This update to the **Spelling Autocorrect** premium plugin removed redundant ‘pre-text’ punctuation, before the word at the start of the sentence being capitalized, from start and replacement of text-pattern.
+
+As a consequence, formatting is now retained when the word is capitalized, as expected.
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.3 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following updates:
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.3 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+[[improvements]]
+== Improvements
+
+{productname} 6.3 also includes the following improvements
+
+=== The Text color and Background color menus now display a checkmark denoting the currently selected color.
```

Comment:
I think the issue here is that there are two issues here, but they are really just one. One improvement can't exist without the other. Should we really have two issues?

Reply:
The anwer to _why are there two entries for this?_, at least from our workflow perspective, is simple.

Although both entries are drawn from the same ticket, [Tiny-9283](https://ephocks.atlassian.net/browse/TINY-9283), there are two entries in the [TinyMCE ChangeLog](https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/CHANGELOG.md).

1. The write-up in the **Changes** section: *Keyboard navigation in the Text color and Background color menus now starts on the currently selected color if that color is present*.
This is expanded from [Colorswatch keyboard navigation now starts on currently selected color if present in the colorswatch. #TINY-9283](https://github.com/tinymce/tinymce/blob/5a0b117e9afc1b20f91daa8308e5f9f5cd44ac81/modules/tinymce/CHANGELOG.md?plain=1#L41)

2. The write-up in the Improvments setion: *The Text color and Background color menus now display a checkmark denoting the currently selected color*.
This is expanded from [Colorswatch now displays a checkmark for selected color. #TINY-9283](https://github.com/tinymce/tinymce/blob/5a0b117e9afc1b20f91daa8308e5f9f5cd44ac81/modules/tinymce/CHANGELOG.md?plain=1#L46)

Because our remit is to do a full write-up of each listed change in the ChangeLog.

This isn’t an iron law, but absent timely information showing why, for example, two ChangeLog entries are better explained in one write-up, we treat each ChangeLog entry as if it is reasonable and important that it exists as a discrete entry.